test/e2e/load.go

```diff
@@ -94,17 +81,45 @@ var _ = Describe("Load capacity", func() {
 
 	type Load struct {
 		podsPerNode int
+		image       string
+		command     []string
+		skip        bool
+		deleteNs    bool
 	}
 
 	loadTests := []Load{
-		{podsPerNode: 30},
+		// The container will consume 1 cpu and 512mb of memory.
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
```

Comment:
Pods consume 3 cores and 1.5gb mem on each node by average, while nodes have 1 core and 3.8gb mem. Seems like quite reasonable load. We definitely don't want to use more memory that is available on the node. Another option is to run more pods with less resources consumed.